debian/cocaine-v12-runtime.init

```diff
@@ -14,7 +14,7 @@
 # Author: Andrey Sibiryov <kobolog@yandex-team.ru>
 
 PATH=/sbin:/bin:/usr/sbin:/usr/bin
-NAME=cocaine-runtime
+NAME=cocaine-v12-runtime
```

Comment:
1. Workflows were introduced in conductor for this exact reason - to handle non-standart package isnt in all repos and/or should be dmoved differently in different repos. I suggest you to try to use that stuff, even it's unclear and vague in a way.
2.  Have you addressed Jay on this problem? Because this kind of "workaround" looks to me as an ugly-dirty-fucking-hack.
¯\_(ツ)_/¯